models/task/task.go

```diff
@@ -178,8 +178,8 @@ func (u *TaskStruct) Update(repo *repository.RepositoryStruct, OauthToken *sql.N
 	return true
 }
 
-// lastに追加する場合にはprevToTaskIdをnullで渡す
-func (u *TaskStruct) ChangeList(listId int64, prevToTaskId *int64, repo *repository.RepositoryStruct, OauthToken *sql.NullString) bool {
+// lastに追加する場合にはprevToTaskIDをnullで渡す
```

Comment:
__[golint]__
* [golint] comment on exported method TaskStruct.ChangeList should be of the form "ChangeList ..."